### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/layout_moyo-header/
https://pavel-gutsal.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,68 @@
...
+        class="header__a" href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+          class="logoImg"
+          >
+      </a>
+      <nav
+        class="nav"
+      >
```
Comment (ArturErmolenko) [inline src/index.html:25]:
```suggestion
      <nav class="nav">
```

File: src/index.html [hunk 2]
```diff
@@ -8,7 +8,68 @@
...
+    <header 
+      class="header"
+    >
+      <a 
+        class="header__a" href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+          class="logoImg"
+          >
```
Comment (ArturErmolenko) [inline src/index.html:21]:
```suggestion
        <img 
          src="./images/logo.png" 
          alt="logo"
          class="logoImg"
        >
```

File: src/index.html [hunk 3]
```diff
@@ -8,7 +8,68 @@
...
+      </a>
+      <nav
+        class="nav"
+      >
+        <ul class="nav__ul">
+          <li class="nav__list">
+            <a class="is-active nav__a" href="#apple">
```
Comment (ArturErmolenko) [inline src/index.html:28]:
```suggestion
            <a class="is-active nav__link" href="#apple">
```

File: src/index.html [hunk 4]
```diff
@@ -8,7 +8,68 @@
...
+              apple
+            </a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__a" href="#samsung">
```
Comment (ArturErmolenko) [inline src/index.html:33]:
```suggestion
            <a class="nav__link" href="#samsung">
```

File: src/index.html [hunk 5]
```diff
@@ -8,7 +8,68 @@
...
+              samsung
+            </a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__a" href="#smartphones">
```
Comment (ArturErmolenko) [inline src/index.html:38]: same `nav__a `its a bad example use `nav__link`

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Good job 🔥 
Just fix comment

**Comment by pavel-gutsal** [comment]
Second commit
all highlighted issues are fixed

**Pull request closed without merging**